src/components.js

```diff
@@ -4,15 +4,22 @@ import { ContainerMixin } from './ContainerMixin';
 export const SlickList = {
   name: 'slick-list',
   mixins: [ ContainerMixin ],
+  props: {
+    tag: {
+      type: String,
+      default: 'div',
+    }
+  },
   render (h) {
-    return h('div', this.$slots.default);
+    return h(this.$props.tag || 'div', this.$slots.default);
```

Comment:
Two more nits: 
`|| 'div'` is redundant.
`this.$props.tag` can be `this.tag`

Reply:
Sorry - I accidentally hit undo... feeling a bit under the weather today! Thanks for tip about just tag.